lst_scripts/onsite_mc_r0_to_dl1.py

```diff
@@ -213,13 +216,16 @@ def main(input_dir, config_file=None, train_test_ratio=0.25, random_seed=42, n_f
             else:  # flag_full_workflow == True !
                 cmd = 'sbatch --parsable -e {} -o {} {} {}'.format(jobe, jobo, base_cmd, os.path.join(dir_lists, file))
 
-                jobid = os.popen(cmd).read().split('\n')
+                jobid = os.popen(cmd).read().strip('\n')
+                jobids_r0_dl1.append(jobid)
 
                 # Fill the dictionaries if IN workflow mode
-                jobid2cmd[jobid] = cmd
-                jobid2outfile[jobid] = jobo
-                jobid2errfile[jobid] = jobe
-                jobid2partype[jobid] = DL0_DATA_DIR.split('/')[-2]  # Hardcoded, maybe if with 4 elif s ?
+                jobid2log[jobid] = {}
+                jobid2log[jobid]['particle'] = DL0_DATA_DIR.split('/')[-2]  # Hardcoded, maybe if with 4 elif s ?
```

Comment:
you can also look for the words `gamma`, `gamma_diffuse`, `proton`, `electron` in the path...

Reply:
Note: careful with `gamma` and `gamma_diffuse`...